### PR TITLE
Adding mapping for customer_first_reply

### PR DESCRIPTION
**Pull request by alexshultsg**
#### Why?
customer_first_reply filed is missing in Java API

#### How?
By adding mapping from json

### CODE REVIEW

**Comment by alexshultsg** [comment]
https://github.com/intercom/intercom-java/issues/262

File: intercom-java/src/main/java/io/intercom/api/CustomerFirstReply.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
+package io.intercom.api;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@SuppressWarnings("UnusedDeclaration")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CustomerFirstReply extends TypedData {
```
Comment (mmartinic) [inline intercom-java/src/main/java/io/intercom/api/CustomerFirstReply.java]: Could you please reformat this class to use spaces for indentation? It's using a mix of tabs and spaces currently

File: intercom-java/src/main/java/io/intercom/api/Conversation.java [hunk 2]
```diff
@@ -214,6 +214,9 @@ private static boolean isAdminQuery(Map<String, String> params) {
 
     @JsonProperty("conversation_message")
     private ConversationMessage conversationMessage;
+    
+    @JsonProperty("customer_first_reply")
+    private CustomerFirstReply customerFirstReply;
```
Comment (mmartinic) [inline intercom-java/src/main/java/io/intercom/api/Conversation.java]: I think you need to add a public getter for `customerFirstReply` here https://github.com/intercom/intercom-java/blob/3e5302cacc7682544d49f41ad9bb0b19f4983137/intercom-java/src/main/java/io/intercom/api/Conversation.java#L300

Could you also add `customerFirstReply` to `hashCode` and `equals`?

**Comment by alexshultsg** [comment]
@mmartinic Done, I also added a unit test. Thank you

**Comment by mmartinic** [comment]
@alexshultsg thanks! I'll take a look

**Comment by alexshultsg** [comment]
@mmartinic  For some reason build are failing with some sort of permission error. could you, please, look at them? Thanks

**Pull request merged**